code/modules/shuttle/on_move.dm

```diff
@@ -396,10 +396,32 @@ All ShuttleMove procs go here
 
 	. = ..()
 
+/obj/docking_port/mobile/fob/onShuttleMove(turf/newT, turf/oldT, rotation, list/movement_force, move_dir, old_dock)
+	. = ..()
+
+	//Unload the planet
+	var/obj/docking_port/stationary/S = old_dock
+	for(var/datum/star_system/SYS in SSstarmap.star_systems)
+		var/datum/planet/PL = SYS.get_planet_for_z(oldT.z)
+		if(S.name == "FOB Landing Zone" && SSstarmap.current_planet.name != PL.name)
+			PL.no_unload_reason = ""
+			PL.do_unload()
```

Comment:
On second thought I'm not sure about unloading immediately, I'll need to check on this later if do_unload is check_ticked.

Reply:
So what do you want me to do then?

Reply:
comment it out for now and leave that bit to another pr. It'll be unloaded anyway if the ship moves.